dotnet/samples/04.e.twentyQuestions/GameTurnState.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.TeamsAI.State;
+
+namespace TwentyQuestions
+{
+    /// <summary>
+    /// Strongly type the turn state
+    /// </summary>
+    public class GameTurnState : TurnState
```

Comment:
We should add these state properties to `ConversationState`. I doubt this is being persisted in storage. Check how I'm setting up the turn state in Echo bot: https://github.com/microsoft/teams-ai/pull/282/files#diff-a2030ddeec5931670f5a23ecc7d23da473bb4ffb4c6fc50903ed3c6073084638

Reply:
Ah yes, updated to use `ConversationState`.
However, after using `TurnState<ConversationState, StateBase, TempState>`, [code here](https://github.com/microsoft/teams-ai/blob/DOTNET/dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Prompt/PromptManager.cs#L182) (`turnState as object is TurnState defaultTurnState`) always returns false and I need to explicitly add "*input*" variable. Is it expected?

Reply:
The should've been removed when I refactored the turn state. The temp state values (like "input") should be available to use in the prompt template. I pushed a fix: https://github.com/microsoft/teams-ai/pull/286.

Reply:
Updated